### PR TITLE
Master wowl design fixes

### PR DESCRIPTION
**Pull request by stefanorigano**
Solves several post-merge owl design issues

pad: https://pad.odoo.com/p/wowl-bugs
task: 2579123

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/legacy/js/views/list/list_renderer.js [hunk 1]
```diff
@@ -209,7 +209,8 @@ var ListRenderer = BasicRenderer.extend({
         const decorations = {};
         for (const [key, expr] of Object.entries(node.attrs)) {
             if (DECORATIONS.includes(key)) {
-                const cssClass = key.replace('decoration', 'text');
+                let cssClass = key.replace('decoration', 'text');
+                cssClass == 'text-bf' ? cssClass = 'font-weight-bold' : cssClass == 'text-it' ? cssClass = 'font-italic' : cssClass;
```
Comment (stefanorigano) [inline addons/web/static/src/legacy/js/views/list/list_renderer.js:213]: Hello @aab-odoo , 
I'm not very proud of these lines but I didn't find any other way to apply the fix keeping changes low... any suggestions?

Comment (aab-odoo) [reply addons/web/static/src/legacy/js/views/list/list_renderer.js:213]: I suggest to re-introduce the two css rules :D It's 6 lines of code including the brackets

Comment (aab-odoo) [reply addons/web/static/src/legacy/js/views/list/list_renderer.js:213]: They were defined globally, which is probably a mistake, so I don't see any problem if you decide to scope them by .o_list_view (in list_view.scss)

File: addons/web/static/src/legacy/js/views/list/list_renderer.js [hunk 2]
```diff
@@ -209,7 +209,14 @@ var ListRenderer = BasicRenderer.extend({
         const decorations = {};
         for (const [key, expr] of Object.entries(node.attrs)) {
             if (DECORATIONS.includes(key)) {
-                const cssClass = key.replace('decoration', 'text');
+                let cssClass = key.replace('decoration', 'text');
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/js/views/list/list_renderer.js:212]: let cssClass;

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Comment by aab-odoo** [comment]
Thanks for the work @stefanorigano and sorry for the delay.

**Pull request closed without merging**